server/boot/challenge.js

```diff
@@ -515,7 +515,8 @@ module.exports = function(app) {
           }
           var view = challengeView[data.challengeType];
           if (data.id) {
-            res.cookie('currentChallengeId', data.id);
```

Comment:
Line is still there, it was just changed with an additional parameter. 

`+            res.cookie('currentChallengeId', data.id, {
 +              expires: new Date(2147483647000)});` 

Not sure what you are asking, "??" is a very unhelpful and ambiguous comment, if you could expand.